BUILDING.md

```diff
@@ -120,6 +110,17 @@ for more information.
 Note that the above requires that `python` resolve to Python 2.6 or 2.7
 and not a newer version.
 
+* On macOS you may want to setup [firewall rules](tools/macosx-firewall.sh)
+to avoid popups asking to accept incoming network connections when running tests:
+
+If the path to your build directory contains a space, the build will likely fail.
```

Comment:
Hi, @joeldart! Welcome and thanks for this! This line about spaces in the directory should remain where it was above. Moving the other ones seems good to me. Can you move this line back?